wordpress-mobile/WordPress-iOS
# Migrates people service remote + models to WordPressKit

Fixes #7317  

This PR:
* Creates a new  `RemotePerson` model in the WPKit framework, any non-remote functionality was moved into the main app's CD model (`ManagedPerson.swift`)
* Migrates the `PeopleServiceRemote` to the WPKit framework
* Migrates related unit tests (`PeopleServiceRemoteTests.swift`) + mock data to WPKit framework
* Migrates `Gravatar.swift` to the WPShared framework, and relocates `downloadGravatar` func to `UIImageView+Gravatar.swift`
* Migrates related unit tests to WPShared framework
* Updates related imports/refs in main WordPress project

Ref #7301

**To test:**
1. Clean DerivedData.
2. Update CocoaPods repo with pod repo update.
3. Install the pods with pod install.
4. Run _all_ unit tests and make sure they are green.
5. Build & run the app:
a. Verify You can view/edit followers on .com sites
b. Verify You can view/edit viewers on private .com sites
c. Verify You can view/edit users on .com sites
d. Verify you can add new users on .com sites

Needs review: @astralbodies 🙇
